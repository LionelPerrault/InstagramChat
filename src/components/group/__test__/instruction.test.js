import React from 'react'
import { create } from 'react-test-renderer'
import GroupInstruction from '../instruction'

describe('GroupInstruction Component', () => {
  it('should match snapshot', () => {
    const tree = create(<GroupInstruction />).toJSON()
    expect(tree).toMatchSnapshot()
  })

  it('should match snapshot and hide buttons', () => {
    const tree = create(<GroupInstruction showBtns={false} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
