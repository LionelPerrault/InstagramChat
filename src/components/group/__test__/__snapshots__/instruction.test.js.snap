// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GroupInstruction Component should match snapshot 1`] = `
<div
  className="sabout_one"
>
  <img
    src="/images/tree.png"
  />
  <div
    className="sabout_one_info"
  >
    <span>
      Update or edit group to make it look more attractive
    </span>
    <Link
      className="sec_btn"
      to="/"
    >
      Update group
    </Link>
    <Link
      className="pri_btn"
      to="/edit-profile"
    >
      Edit group
    </Link>
  </div>
</div>
`;

exports[`GroupInstruction Component should match snapshot and hide buttons 1`] = `
<div
  className="sabout_one"
>
  <img
    src="/images/tree.png"
  />
  <div
    className="sabout_one_info"
  >
    <span>
      Update or edit group to make it look more attractive
    </span>
  </div>
</div>
`;
